x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/policy_details_with_security_essentials.cy.ts

```diff
@@ -16,7 +16,10 @@ describe.skip(
     tags: ['@serverless'],
     env: {
       ftrConfig: {
-        productTypes: [{ product_line: 'security', product_tier: 'essentials' }],
+        productTypes: [
+          { product_line: 'security', product_tier: 'essentials' },
+          { product_line: 'endpoint', product_tier: 'essentials' },
```

Comment:
same here: this change makes the test invalid - it should not be committed.

Reply:
Maybe try putting the same entry twice instead (the `product_line: 'security'` one). The goal of the test is to ensure `product_line: 'endpoint'` is not running.